packages/web-app/src/pages/DocumentDetails/transformers.js

```diff
@@ -31,6 +31,15 @@ export const makeOverview = data => ({
   summary: pipe(propOr([], 'descriptions'), head, propOr('', 'text'))(data)
 });
 
+export const getAuthor = author => ({
+  id: author?.id,
+  fullName: author?.nickname || author?.name || author?.surname || 'Unknown',
+  name: author?.name || 'Unknown',
+  nickname: author?.nickname || 'Unknown',
+  surname: author?.surname || 'Unknown',
+  url: `/ui/persons/${author?.id}`
+});
```

Comment:
À refactoriser dans un fichier à part et à remplacer partout où c'est utilisé.

Reply:
Je réitère ce commentaire en le précisant. Il faudrait mettre cette fonction dans le dossier `src/util`. Ça sert à pleins de transformers, on ne peut pas le laisser dans `DocumentDetails/transformers.js`.